src/bindings/python/src/openvino/runtime/opset1/ops.py

```diff
@@ -67,7 +67,15 @@ def add(
     auto_broadcast: str = "NUMPY",
     name: Optional[str] = None,
 ) -> Node:
-    """Return node which applies f(x) = A+B to the input nodes element-wise."""
+    """Return node which applies f(A,B) = A+B to the input nodes element-wise.
+
+    :param left_node: The first input node for add operation.
+    :param right_node: The second input node for add operation.
+    :param auto_broadcast: The type of broadcasting specifies rules used for
```

Comment:
One idea to consider is to add information about default value like `:param auto_broadcast: The type of broadcasting specifies rules used for auto-broadcasting of input tensors. Defaults to "NUMPY"`